### DISCUSSION TITLE
Futbol ve hileli para sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#309] [Zeka Sorusu]
Her mevkiden 2 futbolcu olmak üzere toplam 22 futbolcu takım seçmelerine katılmaktadır. Aynı mevkilerde oynayan oyuncular arasında yazı tura atılarak 11 kişilik takım belirlenecektir. Antrenörün elinde biri hatalı (yazı tura gelme olasılığı eşit olmayan) diğeri ise hatasız olan ve aynı görüntüye sahip iki adet para vardır. Bu iki para kullanılarak adil bir kura yöntemi nasıl uygulanabilir?

**Comment by Sadik1Dev** [comment]
İki parayı da atar ve her ikisinde de aynı yüz gelirse bunu geçerli sonuç kabul edip ona göre karar verir.

**Comment by slos8888** [comment]
İki para da atılır ve ikisi de aynı yüze denk gelirse alım yapılır.
